PWGEM/Dilepton/Utils/EMTrack.h

```diff
@@ -57,6 +57,7 @@ class EMTrack
   float px() const { return fPt * std::cos(fPhi); }
   float py() const { return fPt * std::sin(fPhi); }
   float pz() const { return fPt * std::sinh(fEta); }
+  float e() const { return std::sqrt(std::pow(fPt * std::cosh(fEta), 2) + std::pow(fMass, 2)); } // e2 = p2 + m2
```

Comment:
You could just use `std::hypot` or `RecoDecay::e`. Btw, all these calculations are available as `RecoDecay` methods.